LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDevice.cs

```diff
@@ -213,6 +213,8 @@ public LoRaRegionType LoRaRegion
 
         public StationEui LastProcessingStationEui => this.lastProcessingStationEui.Get();
 
+        private readonly TimeSpan twinUpdateTimeout = TimeSpan.FromSeconds(10);
```

Comment:
nit: we can make this a static. Can you also try to move it to the private fields? I know it's already a bit a mess, but there are some groups in there.

Reply:
Made internal static for reusal with 55b1819